diff --git a/toolchain/apple/toolchain.gni b/toolchain/apple/toolchain.gni
index 0c0c8d9b1..f5c794ff5 100644
--- a/toolchain/apple/toolchain.gni
+++ b/toolchain/apple/toolchain.gni
@@ -210,7 +210,7 @@ template("single_apple_toolchain") {
 
     # Specify an explicit path for the strip binary.
     _strippath = invoker.bin_path + "strip"
-    _installnametoolpath = "${prefix}llvm-install-name-tool"
+    _installnametoolpath = "${prefix}install_name_tool"
     linker_driver_args += " -Wcrl,strippath,${_strippath} -Wcrl,installnametoolpath,${_installnametoolpath}"
     _enable_dsyms = enable_dsyms
     _save_unstripped_output = save_unstripped_output
